test/Microsoft.AspNet.Authentication.Test/OAuthBearer/OAuthBearerMiddlewareTests.cs

```diff
@@ -27,6 +26,8 @@ public class OAuthBearerMiddlewareTests
         {
             var server = CreateServer(options =>
             {
+                options.AutomaticAuthentication = true;
```

Comment:
Well the behavior we are trying to decide is whether you need to ask for the bearer token via Authenticate, or whether its there automatically even if you don't ask for it (via policy for example)  We were going to make Cookie passive too, but that broke down quickly